text/3453-f16-and-f128.md

```diff
@@ -0,0 +1,124 @@
+- Feature Name: `f16_and_f128`
+- Start Date: 2023-07-02
+- RFC PR: [rust-lang/rfcs#3453](https://github.com/rust-lang/rfcs/pull/3453)
+- Rust Issue: [rust-lang/rfcs#2629](https://github.com/rust-lang/rfcs/issues/2629)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes adding new floating point types `f16` and `f128` into the core language and standard library.
```

Comment:
```suggestion
This RFC proposes adding new IEEE-compliant floating point types `f16` and `f128` into the core language
and standard library. We will provide a soft float implementation for all targets, and use hardware support
where possible.
```

Just easier to tell at a glance that (1) we won't be breaking IEEE compliance, (2) we do plan to support all targets